composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "3.* || 4.*"
+        "symfony/framework-bundle": "3.* || 4.* || 5.*"
```

Comment:
Do you know by any chance what is the smallest Symfony major version that requires your changes? 4.*? or maybe 5.*?

Reply:
I haven't tested in SF4, but after some digging, AbstractExtension from Twig wasn't available until vsersion 2.4.
framework-bundle 4.X requires Twig ~1.34|~2.4. So I think it could work on framework 4, to be sure I would say framework 5.X.